drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java

```diff
@@ -343,12 +343,13 @@ protected void setSaveEnabled(boolean toSet) {
 
     @Override
     protected Supplier<ScenarioSimulationModel> getContentSupplier() {
-        return scenarioSimulationEditorPresenter.getContentSupplier();
+        return () -> scenarioSimulationEditorPresenter.getModel();
     }
 
     @Override
     protected void save(final String commitMessage) {
-        synchronizeColumnsDimension();
```

Comment:
@gitgabrio There was a reason to remove this method an duplicate its logic in `save()` and `getBeforeSaveAndRenameCommand()` ?

Reply:
@gitgabrio Ok, found it, you moved the logic in the `Wrapper`